targets/targets.json

```diff
@@ -14793,6 +14793,61 @@
             "3701"
         ]
     },
+    "NUCLEO_G474RE": {
+        "inherits": [
+            "FAMILY_STM32"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M4F",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSI",
```

Comment:
I think using USE_PLL_HSE_XTAL would be better ?

Reply:
Agreed, I updated this. See 4f3a1efec99c9b36784d17828050b0746b045352